<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>326</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>385</width>
    <height>326</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>385</width>
    <height>326</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NTB Data Collection Automation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QProgressBar" name="masterProgressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>244</width>
          <height>140</height>
         </rect>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>251</width>
           <height>141</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButtonDownloadForDay">
       <property name="text">
        <string>Download For Day</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButtonDownloadForMonth">
       <property name="text">
        <string>Download For Month</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QDateEdit" name="daySelector"/>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButtonDownloadForYear">
       <property name="text">
        <string>Download For Year</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QDateEdit" name="yearSelector">
       <property name="displayFormat">
        <string>yyyy</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QDateEdit" name="monthSelector">
       <property name="displayFormat">
        <string>M/yyyy</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButtonSelectAll">
         <property name="text">
          <string>Select All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDisselectAll">
         <property name="text">
          <string>Disselect All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNTB_Datacollection_Automation">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="openConfFileButton"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="openLogFileButton"/>
   </widget>
   <addaction name="menuNTB_Datacollection_Automation"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openConfFileButton">
   <property name="text">
    <string>Open Configuration File</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSetup_Scheduled_Task">
   <property name="text">
    <string>Setup Scheduled Task</string>
   </property>
  </action>
  <action name="actionReload_Configuration_File">
   <property name="text">
    <string>Reload Configuration File</string>
   </property>
  </action>
  <action name="openLogFileButton">
   <property name="text">
    <string>Open Log</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
